src/Twig/SourceCodeExtension.php

```diff
@@ -108,9 +117,13 @@ private function getCallableReflector(callable $callable): \ReflectionFunctionAb
             return $r->getMethod('__invoke');
         }
 
+        // @phpstan-ignore-next-line
```

Comment:
instead of ignoring all errors on next line, it is better to keep a given error in the baseline, because other errors would still be reported on that line if we introduce more.

Reply:
Created baseline